From ea02c30d4d4e70f79735e1c4cbfa77258c8b9dca Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 25 Mar 2021 18:40:51 +0000
Subject: [PATCH] Revert "drm/vc4: Add configuration for BCM2711 DSI1."

This reverts commit 10faa28e1474e6cf33cb0809d2a6d5b7a8351a28.

Revert in order to take the cleaner upstream version.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/gpu/drm/vc4/vc4_dsi.c | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_dsi.c b/drivers/gpu/drm/vc4/vc4_dsi.c
index bac5e07c05de..ce4db68827c8 100644
--- a/drivers/gpu/drm/vc4/vc4_dsi.c
+++ b/drivers/gpu/drm/vc4/vc4_dsi.c
@@ -1308,11 +1308,6 @@ static const struct drm_encoder_helper_funcs vc4_dsi_encoder_helper_funcs = {
 static const struct of_device_id vc4_dsi_dt_match[] = {
 	{ .compatible = "brcm,bcm2835-dsi0", (void *)(uintptr_t)0 },
 	{ .compatible = "brcm,bcm2835-dsi1", (void *)(uintptr_t)1 },
-	/*
-	 * Use 2 so that it uses the DSI1 register layout, but not DMA
-	 * workaround
-	 */
-	{ .compatible = "brcm,bcm2711-dsi1", (void *)(uintptr_t)2 },
 	{}
 };
 
@@ -1500,8 +1495,8 @@ static int vc4_dsi_bind(struct device *dev, struct device *master, void *data)
 		return -ENODEV;
 	}
 
-	/* DSI1 on BCM2835/6/7 has a broken AXI slave that doesn't respond to
-	 * writes from the ARM.  It does handle writes from the DMA engine,
+	/* DSI1 has a broken AXI slave that doesn't respond to writes
+	 * from the ARM.  It does handle writes from the DMA engine,
 	 * so set up a channel for talking to it.
 	 */
 	if (dsi->port == 1) {
-- 
2.17.1

